lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/repeat-fraud-check.yaml

```diff
@@ -45,6 +48,32 @@ states:
 
   # Journey States
 
+  CONFIRM_DETAILS:
+    response:
+      type: page
+      pageId: confirm-your-details
+      context: coi
```

Comment:
Looking at Sam's PR for this page, I don't think we need to include this context. The page will only have one form.

Reply:
@Wynndow yes thats right - sorry I wrongly told @thebauSoftwire to add it when I as under the impression we would be editing an existing page